### PR TITLE
docs: add summit callouts

### PR DESCRIPTION
**Pull request by Meschreiber**
Adds a callout/links to relevant GraphQL summit workshops

### CODE REVIEW

File: docs/source/configuration/authn-jwt.mdx [hunk 1]
```diff
@@ -4,6 +4,12 @@ subtitle: Restrict access to credentialed users and systems
 description:  Protect sensitive data by enabling JWT authentication in the Apollo GraphOS Router. Restrict access to credentialed users and systems.
 ---
 
+<SummitCallout
```
Comment (shorgi) [inline docs/source/configuration/authn-jwt.mdx:7]: `<SummitCallout>` and `<PremiumFeature>` have different icon + text layout, and they don't look consistent together [here](https://deploy-preview-5830--apollo-router-docs.netlify.app/configuration/authn-jwt). Move the summit icon?

File: docs/source/configuration/distributed-caching.mdx [hunk 2]
```diff
@@ -150,7 +156,7 @@ The `ttl` option defines the default global expiration for Redis entries. For AP
 
 ### Namespace
 
-When using the same Redis instance for multiple purposes, the `namespace` option defines a prefix for all the keys defined by the router.
+When using the same Redis instance for multiple puposes, the `namespace` option defines a prefix for all the keys defined by the router.
```
Comment (shorgi) [inline docs/source/configuration/distributed-caching.mdx:159]:
```suggestion
When using the same Redis instance for multiple purposes, the `namespace` option defines a prefix for all the keys defined by the router.
```

**Review by shorgi — APPROVED** [review]

File: docs/source/configuration/authn-jwt.mdx (hunk 1 above)
Comment (Meschreiber) [reply docs/source/configuration/authn-jwt.mdx:7]: The <SummitCallout> follows the styling of Tips/Cautions/etc.

See below on the same page:

image


I'm personally fine with it, but we can definitely try to align all callout styles in a different PR!

**Review by goto-bus-stop — APPROVED** [review]
🥳

**Pull request merged**